src/export.rs

```diff
@@ -95,7 +110,7 @@ pub fn export_mmp<W: Write>(
 }
 
 /// Export an mmp file for a given proof tree.
-pub fn export_mmp_proof_tree<W: Write>(
+fn export_mmp_proof_tree<W: Write>(
```

Comment:
I'd like this function to be part of the API of the library, for the case where a proof assistant builds a proof and would like to export an MMP file with it. The proof format used in the API is then the `ProofTreeArray`.

This function can be attached to the `Database` object, though, which allows to drop the pass results in the parameters.